Docs/Request_Overrides.md

```diff
@@ -181,3 +181,67 @@ Example:
      ...
 }
   ```
+## screenshot:
+
+### Request override
+- The `screenshot` request override is used to take a screenshot of the current screen of the platform. This function has to be added in config module `requestModules/fcs.js` file.
+
+#### Request format for screenshot request override function:
+
+```javascript
+{
+  method: 'screenshot',
+  params: {
+    validations: []
+  }
+}
+```
+
+**Examples:**
+
+**Example 1:** Passing validation objects to request override to validate the image with label `auth` with confidence 60 against the screenshot response.
+
+```javascript
+{
+  method: 'screenshot',
+  params: {
+    validations: [
+      {
+        "type": "image",
+        "label": "auth",
+        "confidence": 60
+      }
+    ]
+  }
+}
+```
+
+### Response override
+
+- The `screenshot` response override is used to validate the screenshot response against the validation object passed in the request override function. This function is added in the config module `responseModules/fcs.js` file.
+
+#### Return response format of screenshot response override function:
+- In the response below, `status` is the key that indicates the overall status of the screenshot validation. `validations` is an array of objects that holds the status of each validation object.
+- `status` is the required field to determine the status of the validation.
+
+```javascript
+{
+  status: "pass/fail",
+  validations: [
+    {status: "pass/fail"},
+  ]
+}
+```
+
+**Examples:**
+
+**Example 1:** 
+
+```javascript
+{
+  status: "pass",
+  validations: [
+    {status: "pass"}
+  ]
```

Comment:
give example of failure case also and give one liner to each example

Reply:
addressed the comment